google/auth/crypt/__init__.py

```diff
@@ -65,9 +67,11 @@
 RSASigner = rsa.RSASigner
 RSAVerifier = rsa.RSAVerifier
 
-if es256 is not None:  # pragma: NO COVER
-    ES256Signer = es256.ES256Signer
-    ES256Verifier = es256.ES256Verifier
+if es is not None:  # pragma: NO COVER
+    ES256Signer = es.EsSigner
+    ES256Verifier = es.EsVerifier
```

Comment:
This is confusing to me. We do have `ES256Verifier`/`ES256Signer` as separate classes in the the es256 module, but instead you're importing the base class with a new name?

I understand they are both set up as aliases, but maybe they're not both needed?

Reply:
Yes, we need to keep them around to avoid breaking existing code. Can we replace these with imports from the `es256.py` file, so we are defining the alias in one place?